### ISSUE TITLE
Hello, this's bullshit! 

### ISSUE THREAD

**Issue by Yoloabdo** [payfort/fort-ios-sdk#2]
I'm not sure what you are guys trying to do here, but this's total bullshit! 

First of all your sdk isn't cocoapods, neither anything else that manages library, and you're forcing devs to go the wrong way by adding library manually!! 

Second: no release notes, updates, fixes!

Third: your API documentation sucks! the colors! the code snippets! like everything! I'm not sure what're you trying to do actually, but this's a huge waste of time, you're there to make developers life easier, not a nightmare!

4th: Hope you take this seriously and start working on making it better, check @stripe as a refrence in code as much as you did in business, thanks!

**Comment by naveedahmad99** [comment]
Yes you are right. Payfort is very bad gateway. @Yoloabdo I checked your medium post and now they changed PKJVFloat Field which is crashing. your solution not working anymore. :-(

**Comment by Yoloabdo** [comment]
a proper documentation won't hurt anyone, but 🤷🏼‍♂️

**Comment by SalahAwadi91** [comment]
You are right, i spend 2 weeks just for integration and its not working yet

**Comment by Dania-Mahmoud** [comment]
Hi Salah / Yoloabdo,

Thanks for raising your comments with us.

Could you please reach us out through **integration@payfort.com** email so we can help you to sort out your technical issues.
